modules/oadp-scheduling-backups.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-scheduling-backups_{context}"]
+= Scheduling backups
+
+You schedule backups by creating a `Schedule` custom resource (CR) instead of a `Backup` CR.
```

Comment:
Should you mention in the assembly that you create the Schedule CR *instead* of the Backup CR?

_You can schedule backups by creating a Schedule CR._